internal/cmd/scraper/scraper.go

```diff
@@ -163,16 +166,44 @@ func Run(cfg *Config) {
 					err,
 				)
 			}
+
+			transportOption, err := integration.TelemetryHarvesterWithInfraTransport(
+				cfg.LicenseKey,
+				cfg.EmitterProxy,
+				cfg.EmitterCaFile,
+				cfg.EmitterInsecureSkipVerify,
+			)
+
+			if err != nil {
+				logrus.Fatalf(
+					"couldn't set emitter with proxy %v CA file %v and insecureSkipVerify %v: %v",
+					cfg.EmitterProxy,
+					cfg.EmitterCaFile,
+					cfg.EmitterInsecureSkipVerify,
+					err,
+				)
+			}
+
 			harvesterOpts := []func(*telemetry.Config){
 				telemetry.ConfigAPIKey(cfg.LicenseKey),
 				telemetry.ConfigBasicErrorLogger(os.Stdout),
-				integration.TelemetryHarvesterWithInfraTransport(cfg.LicenseKey),
+				transportOption,
 				integration.TelemetryHarvesterWithMetricsURL(cfg.MetricAPIURL),
 				integration.TelemetryHarvesterWithHarvestPeriod(hTime),
 			}
 
+			logrus.Debugf("telemetry emitter configured with API endpoint: %s", cfg.MetricAPIURL)
```

Comment:
Outputting the same message twice. Also, this will spam the logs with a lot of `telemetry emitter configured with` messages; coming them into 1 message would be cleaner, like:
`logrus.Debugf("Telemetry emitter options: %#v", cfg)`

Reply:
There is already a `logrus.Debugf("Telemetry emitter options: %#v", cfg)` line so I removed all the debugs.